config/delta-producer/publication-graph-maintainer/config.json

```diff
@@ -137,5 +137,29 @@
     "publisherUri": "http://data.lblod.info/services/delta-producer-publication-graph-maintainer-vendor-management",
     "filesGraph": "http://redpencil.data.gift/id/deltas/producer/vendor-management-deltas",
     "account": "http://data.lblod.info/foaf/account/id/0a3d5be8-6f6b-4c4e-9069-9b7c57a35df3"
-  }
+  },
+    "worship-services-sensitive": {
```

Comment:
The indentation of the "worship-services-sensitive" block should be moved back one tab in order to match with the rest of file.